tests/unit/Core/Saas/SsoServiceTest.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Unit\Core\Saas;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Administration\Login\Config\LoginConfigService;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Sso\SsoService;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+#[CoversClass(SsoService::class)]
+class SsoServiceTest extends TestCase
+{
+    public function testIsSsoShouldReturnTrue(): void
+    {
+        $loginConfigService = new LoginConfigService(
+            [
+                'use_default' => false,
+                'client_id' => 'c6a7ab8a-5c0c-4353-a38a-1b42479ef090',
+                'client_secret' => '42fec3f9-a19b-4796-bce9-cb395a28da9f',
+                'redirect_uri' => 'https://redirect.to',
+                'base_url' => 'https://base.url',
+                'authorize_path' => '/authorize',
+                'token_path' => '/token',
+                'jwks_path' => '/jwks.json',
+                'scope' => 'scope',
+                'register_url' => 'https://register.url',
+            ],
+            'local.host',
+            '/admin'
+        );
+
+        $ssoService = new SsoService($loginConfigService);
+
+        static::assertTrue($ssoService->isSso());
+    }
+
+    public function testIsSsoShouldReturnFalse(): void
+    {
+        // @phpstan-ignore-next-line argument.type
```

Comment:
```suggestion
        // @phpstan-ignore argument.type (please add a comment why we need to ignore)
```

`phpstan-ignore-next-line` works without identifier and will just ignore everything on the next line. which is not wanted